projects/Mallard/src/components/article/html/components/header.ts

```diff
@@ -80,28 +80,27 @@ const shareButtonColor = ({ colors, theme }: CssProps) => {
 }
 
 const threeLines = `
-background-image: repeating-linear-gradient(
-    to bottom,
-    ${color.dimLine},
-    ${color.dimLine} 1px,
-    transparent 1px,
-    transparent 4px
-);
-background-repeat: repeat-x;
-background-position: bottom;
-background-size: 1px 16px;
-content: '';
-display: block;
-height: 16px;
-margin: 0;
+    background-image: repeating-linear-gradient(
```

Comment:
Does the linter complain about the whitespace? Maybe worth a check, but if not this is good to go

Reply:
These are all changes resulting from running the linter and fixing - header.ts was pretty out of sorts before 👍